src/libs/PaymentUtils.js

```diff
@@ -79,6 +79,8 @@ function formatPaymentMethods(bankAccountList, cardList, payPalMeUsername = '',
             accountType: CONST.PAYMENT_METHODS.DEBIT_CARD,
             accountData: _.extend({}, card, {icon}),
             isDefault,
+            errors: card.errors,
```

Comment:
NAB for this PR, but isn't all this code data massaging and thus something we should not be doing?

Reply:
yes, I think there was an issue to move this massaging to the PHP layer

Reply:
Have a link?

Reply:
Ah damn, I remembered wrong it was for policy data https://expensify.slack.com/archives/C02HWMSMZEC/p1656519994560219


I can make an issue to have PHP do the data massaging and for bank accounts as well

Reply:
Issue here: https://github.com/Expensify/Expensify/issues/222615

Reply:
I may be wrong about this, but I understand that the data massaging that we are not allowing is:

Response returns data => massage data => put the data in Onyx

This is not the case here, I think we are reading this from Onyx, then process (massage) it to use it/render. This in IMO is reasonable, as I think expecting to store the data in Onyx with the exact data and structure we will be using for rendering in different places is not necessarily a good idea (we may need different things in different cases for the same data). I.e. I think here we are adding some icon/iconSize prop, this wouldn't make sense for me to come from the backend and getting added in Onyx.